### ISSUE TITLE
Pretrain Inception-RNv2 model

### ISSUE THREAD

**Issue by mbrookhart** [NervanaSystems/ngraph-mxnet#298]
Inception ResnetV2 is available as one of the models in the default imagenet example:
https://github.com/apache/incubator-mxnet/blob/master/example/image-classification/train_imagenet.py 
https://github.com/apache/incubator-mxnet/blob/master/example/image-classification/symbols/inception-resnet-v2.py

Acceptance Criteria:
1) Train model to the default number of epochs and within 10% expected accuracy (on GPU)
2) Upload saved model symbol json and parameters to deepmark
3) Provide a script for testing model accuracy on test data.
